src/play_cljc/gl/core.cljc

```diff
@@ -201,7 +201,7 @@
         uniform-locations (reduce
                             (fn [m uniform]
                               (assoc m uniform
-                                (gl game getUniformLocation program (name uniform))))
+                                (gl game getUniformLocation program ^String (name uniform))))
```

Comment:
Note here as well, there is no `String` in cljs.

Reply:
This one turns out to be unnecessary for avoiding reflection anyways. But what about the other ones like this? Do you want to leave them in, or perhaps wrap them in reader conditionals?